packages/circuit-ui/components/Table/__snapshots__/Table.spec.tsx.snap

```diff
@@ -1839,7 +1839,6 @@ tbody .circuit-11:last-child td {
 >
   <div
     class="circuit-26"
-    height="0px"
```

Comment:
Just noticed that renaming the `height` prop to match the state key actually broke the snapshots, fixing 🔧

Reply:
Fixed in c7c61e9